crates/re_viewer/build.rs

```diff
@@ -144,6 +144,7 @@ fn examples() -> Result<Vec<Example>> {
             examples.push(Example { name, readme });
         }
     }
+    examples.sort_unstable_by(|a, b| a.name.cmp(&b.name));
```

Comment:
I'm assuming the final order impacts how the website renders thing? In which I would expect us to have some kind  of `order`/`weight`/`priority`/whatever field and sort on that instead.

Reply:
This doesn't impact the website, the order there is defined implicitly by how examples are placed in `examples/manifest.yml`. At some point we want to have all the examples rendered, so we'll have to figure out how to make that work